fileversion.go

```diff
@@ -218,3 +240,57 @@ func (self *FileVersion) GetFixedFileInfo() (*FixedFileInfo, error) {
 		FileSubtype: VerFileSubtype(ffi.FileSubtype),
 	}, nil
 }
+
+func (self *FileVersion) GetFileInfoTranslations() ([]FileInfoTranslation, error) {
```

Comment:
Since this is already on the `FileVersion` object, we could call this just `GetTranslations` so as not to be too verbose.